.gitignore

```diff
@@ -33,6 +33,7 @@ __pycache__/
 /scanpy.egg-info/
 /*-env/
 /env-*/
+/setup.py
```

Comment:
This isn't relevant anymore, since there is a `setup.py` file in this PR, right?

Reply:
I added a comment explaining it